src/sentry/workflow_engine/handlers/condition/assigned_to_handler.py

```diff
@@ -0,0 +1,43 @@
+from collections.abc import Sequence
+from typing import Any
+
+from sentry.models.group import Group
+from sentry.models.groupassignee import GroupAssignee
+from sentry.notifications.types import AssigneeTargetType
+from sentry.utils.cache import cache
+from sentry.workflow_engine.models.data_condition import Condition
+from sentry.workflow_engine.registry import condition_handler_registry
+from sentry.workflow_engine.types import DataConditionHandler, WorkflowJob
+
+
+@condition_handler_registry.register(Condition.ASSIGNED_TO)
+class AssignedToConditionHandler(DataConditionHandler[WorkflowJob]):
+    @staticmethod
+    def get_assignees(group: Group) -> Sequence[GroupAssignee]:
+        cache_key = f"group:{group.id}:assignees"
+        assignee_list: Sequence[GroupAssignee] | None = cache.get(cache_key)
+        if assignee_list is None:
+            assignee_list = list(group.assignee_set.all())
+            cache.set(cache_key, assignee_list, 60)
```

Comment:
is 60 in seconds? is that a bit short 🤔

Reply:
🤷🏼

Reply:
the original PR doesn't have much context 😭 oh well. at least we're keeping it the same